bootstrap.py

```diff
@@ -503,6 +504,12 @@ def check_rhn_registration():
     else:
         return False
 
+def enable_repos():
+    for repo in options.enablerepos.split(','):
+      print_running("Enabling repo - %s" % repo)
+      exec_failok("/usr/sbin/subscription-manager repos --enable %s" % repo)
```

Comment:
`sub-man repos --enable <foo>` is quite expensive.
Instead of calling it once for every repo requested, I would suggest to run it once with `--enable foo1 --enable foo2 …`

```python
>>> repos = "foo,bar,baz"
>>> " ".join(['--enable=%s' % i for i in repos.split(',')])
'--enable=foo --enable=bar --enable=baz'
```